ffmpegaudio.cc

```diff
@@ -94,7 +94,11 @@ struct DecoderContext
   QByteArray audioData_;
   QDataStream audioDataStream_;
   AVFormatContext * formatContext_;
+#if LIBAVCODEC_VERSION_MAJOR < 59
```

Comment:
how does the number 59 come from?

Reply:
I'm not the author of PR, but perhaps this could help: https://www.ffmpeg.org/doxygen/2.5/libavcodec_2version_8h_source.html

Reply:
The version was bumped in this commit: https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/85ba17f36dbfde1baeaa47e14d30c337add52c0d